js/main.js

```diff
@@ -20,10 +23,13 @@ function handleSubmit(e) {
     notes: notesInput.value,
     nextEntryId: data.nextEntryId
   };
+  var latestEntry = createEntry(newInput);
+  userEntries.appendChild(latestEntry);
```

Comment:
here you want to use `prepend` instead of append child so that the new entries are loaded ahead of the old entries.